environments/.env.latest

```
@@ -1,22 +1,43 @@
 # Domain and ports config
 FQDN=verificationstaging.komputing.org
+
+# Monitor config
 MONITOR_EXTERNAL_PORT=3000
 MONITOR_PORT=80
+
+# UI config
 UI_EXTERNAL_PORT=1234
 UI_PORT=1234
+
+# Server config
 SERVER_EXTERNAL_PORT=5000
 SERVER_PORT=80
+SERVER_URL=https://verificationstaging.komputing.org/server
+
+# Localchain config
 LOCALCHAIN_PORT=8545
-LOCALCHAIN_EXTERNAL_PORT=8454
+LOCALCHAIN_EXTERNAL_PORT=8545
 LOCALCHAIN_URL=http://localchain:8545
+
+# Repository config
 REPOSITORY_EXTERNAL_PORT=10000
 REPOSITORY_PORT=80
 REPOSITORY_URL=https://contractrepostaging.komputing.org/
+REPOSITORY_PATH=../../data/repository
+DATABASE_PATH=../../data/database/
+
+# S3 config
+AWS_ACCESS_KEY_ID=xxx
+AWS_SECRET_ACCESS_KEY=xxx
+BUCKET_NAME_STAGING=s3://staging-source-verify
 
+# Geth config
+NETWORK_NAME=goerli
+
+# IPFS config
+IPFS_SECRET=xxx
 
 # Other config
-#can be stable for production or latest for development
 TESTING=true
```

Setting TESTING to true make the monitor listen to localchain **only**. 
```suggestion
TESTING=false
```